File: 2017/10/spec/runner.spec.js
import { parse } from "../lib/parser";
import { part1, part2 } from "../lib/runner";

describe("Runner", () => {
  test("part 1 passes tests cases", () => {
    expect(part1([0, 1, 2, 3, 4], [3, 4, 1, 5])).toEqual(12);
  });

  test("part 2 passes tests cases", () => {
  });
});

File: 2020/6/spec/runner_spec.rb
# frozen_string_literal: true

require "runner"
require "parser"

describe Runner do
  it "calculates the correct answer for the test input for part1" do
    expect(Runner.part_1(Parser.parse(sample_input))).to eq(11)
  end

  it "calculates the correct answer for the test input part2" do
    expect(Runner.part_2(Parser.parse(sample_input))).to eq(6)
  end

  def sample_input
    <<~INPUT.strip
      abc

      a
      b
      c

      ab
      ac

      a
      a
      a
      a

      b
    INPUT
  end
end
